x-pack/index.js

```diff
@@ -44,6 +44,6 @@ module.exports = function (kibana) {
     indexManagement(kibana),
     consoleExtensions(kibana),
     notifications(kibana),
-    kueryAutocomplete(kibana),
+    kueryAutocomplete(kibana)
```

Comment:
I like the trailing comma! :/

Reply:
Me too, haha. I just checked out this file from master to revert it. I didn't want a change in an unrelated file to be extra noise